indigo_social/badges.py

```diff
@@ -58,6 +67,25 @@ def revoke(self, user):
         """
         pass
 
+    def send_templated_email(self, template_name, recipient, context, **kwargs):
```

Comment:
This is already implemented in the Notifier class, (see: https://github.com/laws-africa/indigo/blob/ft-annotation-reply-email-notification/indigo_app/notifications.py), no need to redo it here again. 

Pinax Badges send a `badge_awarded` signal that you can receive in the Notifier class and dispatch the email from there, like we do for other event-based notifications. Also, these badge notifications will be used by another part of this project.

You can refer to this, as I have done email notifications for new replies to an annotation: https://github.com/laws-africa/indigo/blob/5f33c59c39b951a9464001edcdb7e9093da26fb8/indigo_app/notifications.py#L143